syntax = "proto3";

// The classifier service definition.
service Classifier {
  // Sends a classification
  rpc GetClassification (ClassificationRequest) returns (ClassificationReply) {}
}

// The request message containing the image.
message ClassificationRequest {
   repeated Image classificationRequest = 1;
}

// The response message containing the classification label
message ClassificationReply {
  repeated ClassificatioMessage classificationResponse = 1;
}

message Image {
  string id = 1;
  bytes chunk_data = 2;
}

message ClassificatioMessage {
  string id = 1;
  string result = 3; 
}


